### PR TITLE
[zh] Sync 1.19.4 release note and 1.20 major release update into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync 1.19.4 release note and 1.20 major release update into Chinese

## Reviewers

@windsonsea 

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.19.x/announcing-1.19.4/index.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+本发布说明描述了 Istio 1.19.3 和 Istio 1.19.4 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **改进** 改进了 `iptables` 锁定功能。新的实现在需要时使用
+  `iptables` 内置锁等待，并在不需要时完全禁用锁定。
+
+- **新增** 添加了门控标志 `ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS`
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19.4/index.md:19]:
```suggestion
- **新增** 添加了门控标志 `ISTIO_ENABLE_IPV4_OUTBOUND_LISTENER_FOR_IPV6_CLUSTERS`，
```

File: content/zh/news/releases/1.19.x/announcing-1.19.4/index.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  用来管理仅 IPv6 的集群额外出站侦听器，以处理 IPv4 NAT 出站流量。
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19.4/index.md:20]:
```suggestion
  在只有 IPv6 的集群中用来管理一个附加的出站侦听器，以处理 IPv4 NAT 出站流量。
```

File: content/zh/news/releases/1.19.x/announcing-1.19.4/index.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  这对于仅 IPv6 集群环境（例如管理仅出口 IPv4 和 IPv6 IP 的 EKS）非常有用。
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19.4/index.md:21]:
```suggestion
  这对于只有 IPv6 的集群环境（例如管理只有 Egress IPv4 以及 IPv6 IP 的 EKS）非常有用。
```

File: content/zh/news/releases/1.19.x/announcing-1.19.4/index.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+- **修复** 修复了 `ServiceEntry` 通配符基于 `glibc` 容器的搜索域后缀 DNS 代理解析的问题。
+  ([Issue #47264](https://github.com/istio/istio/issues/47264))、
+  ([Issue #31250](https://github.com/istio/istio/issues/31250))、
+  ([Issue #33360](https://github.com/istio/istio/issues/33360))、
+  ([Issue #30531](https://github.com/istio/istio/issues/30531))、
+  ([Issue #38484](https://github.com/istio/istio/issues/38484))
+
+- **修复** 修复了如果默认 IP 寻址不是 IPv6，
+  则正在使用 `IstioIngressListener.defaultEndpoint` 的
+  Sidecar 资源无法利用 [::1]:PORT 的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.19.x/announcing-1.19.4/index.md:36]:
```suggestion
  Sidecar 资源无法使用 [::1]:PORT 的问题。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+- /zh/news/announcing-1.20
+- /zh/news/announcing-1.20.0
+---
+
+我们很高兴地宣布 Istio 1.20 发布。这是 2023 年内最后一个 Istio 版本，
+我们要感谢整个 Istio 社区对 1.20.0 版本发布所作出的帮助。
+我们要感谢此版本的几位发布经理：来自 DaoCloud 的 `Xiaopeng Han`、
+来自 Google 的 `Aryan Gupta` 和来自 Tetrate 的 `Jianpeng He`。
+这些发布经理们要特别感谢测试和发布工作组负责人 Eric Van Norman (IBM) 在整个发布周期中提供的帮助和指导。
+我们还要感谢 Istio 工作组的维护者以及广大 Istio 社区，在发布过程中提供及时反馈、
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:18]:
```suggestion
我们还要感谢 Istio 工作组的维护者以及广大 Istio 社区，感谢他们在发布过程中提供及时反馈、
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+审核和社区测试，以及在确保及时发布方面给予的全力支持。
+
+{{< relnote >}}
+
+{{< tip >}}
+Istio 1.20.0 已得到 Kubernetes `1.25` 到 `1.28` 的官方正式支持。
+{{< /tip >}}
+
+## 新特性 {#what-s-new}
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:27]:
```suggestion
## 新特性 {#whats-new}
```
We can follow the en anchors: https://istio.io/latest/news/releases/1.20.x/announcing-1.20/#whats-new

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+### Gateway API
+
+Kubernetes [Gateway API](http://gateway-api.org/)
+是一项旨在为 Kubernetes 带来丰富的服务网络 API
+（类似于 Istio VirtualService 和 Gateway）的举措。
+
+Kubernetes [Gateway API 现已正式发布](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/)
+并且 Istio 已对其提供[全面支持](https://gateway-api.sigs.k8s.io/implementations/#istio)！
+这是整个 Kubernetes 生态系统社区共同努力的结果，并且产生了多种一致性实现
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:37]:
```suggestion
这是整个 Kubernetes 生态系统社区共同努力的结果，并且产生了多种合规实现
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+（包括 [Istio 的完整一致的实现](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/istio-istio.yaml)）。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:38]:
```suggestion
（包括 [Istio 完全合规的实现](https://github.com/kubernetes-sigs/gateway-api/blob/main/conformance/reports/v1.0.0/istio-istio.yaml)）。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 9]
```diff
@@ -0,0 +1,109 @@
...
+
+这标志着一个重要的里程碑，因为 Istio 用户现在可以利用一组稳定
+Gateway API 功能来增强生产环境中的流量管理和入口控制。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:41]:
```suggestion
Gateway API 功能集来增强生产环境中的流量管理和入口控制。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 10]
```diff
@@ -0,0 +1,109 @@
...
+查看 [Gateway API 任务](/zh/docs/tasks/traffic-management/ingress/gateway-api/)并开始使用。
+
+在此版本中，我们还添加了通过 `targetRef` 字段为 Kubernetes
+Gateway API 配置 Istio CRD `AuthorizationPolicy`、`RequestAuthentication`、
+`Telemetry` 和 `WasmPlugin` 的支持。
+
+### 改进后的 ExternalName 服务支持 {#revamped-externalname-service-support}
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:48]:
```suggestion
### 重构对 ExternalName 服务的支持 {#revamped-externalname-service-support}
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 11]
```diff
@@ -0,0 +1,109 @@
...
+
+Istio 1.20 引入了 `ExternalName` 服务的全新更新，与 Kubernetes 行为更加一致。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:50]:
```suggestion
Istio 1.20 引入了针对 `ExternalName` 服务的全新更新，与 Kubernetes 行为更加一致。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 12]
```diff
@@ -0,0 +1,109 @@
...
+此更新简化了 `ServiceEntry` 定义并增强了 Istio 处理 DNS 条目的能力。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:51]:
```suggestion
本次变更简化了 `ServiceEntry` 定义并增强了 Istio 处理 DNS 条目的能力。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 13]
```diff
@@ -0,0 +1,109 @@
...
+### 扩展对网络 WasmPlugin 的支持 {#expanded-support-for-network-wasmplugin}
+
+通过使用新类型 `NETWORK` 对网络 WasmPlugin 进行支持，
+Istio 的可扩展性进一步扩大。
+
+### TCP 元数据交换增强 {#tcp-metadata-exchange-enhancements}
+
+Istio 1.20 带来了两项关键更新用于帮助控制 TCP 元数据交换：
+
+- **回退元数据发现** Istio 现在可以使用备份方法来收集元数据。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:68]:
```suggestion
- **回退元数据发现**：Istio 现在可以使用备份方法来收集元数据。
```
en 漏了个冒号

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+  要使用此功能，请在代理中开启 `PEER_METADATA_DISCOVERY`，
+  并在控制平面中开启 `PILOT_ENABLE_AMBIENT_CONTROLLERS`。
+- **ALPN 令牌控制**：控制平面有一个名为`PILOT_DISABLE_MX_ALPN` 的新设置。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:71]:
```suggestion
- **ALPN 令牌控制**：针对控制平面推出一个名为 `PILOT_DISABLE_MX_ALPN` 的新设置。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+  这可以让您停止使用服务之间相互通信通常需要的特定令牌 `istio-peer-exchange`。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:72]:
```suggestion
  这可以让您停止使用服务之间相互通信时正常所需的特定令牌 `istio-peer-exchange`。
```

File: content/zh/news/releases/1.20.x/announcing-1.20/_index.md [hunk 16]
```diff
@@ -0,0 +1,109 @@
...
+
+Istio 在 OpenShift 集群上的安装过程已得到简化，
+无需向 Istio 和应用程序授予 `anyuid` SCC 权限。
+
+### `istioctl` 命令的增强 {#enhancements-to-the-istioctl-command}
+
+为 istioctl 命令添加了许多增强功能，包括：
+
+- 如果未将 Pilot 监控端口设置为 `15014`，现在该问题可被自动检测到。
+- 添加了 `istioctl dashboard proxy` 命令来显示不同类型代理的管理 UI，包括 Envoy、Ztunnel、Waypoint。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20/_index.md:99]:
```suggestion
- 添加了 `istioctl dashboard proxy` 命令来显示 Envoy、Ztunnel、Waypoint 等各种代理的管理 UI。
```

**Review by windsonsea — COMMENTED** [review]
Thanks for your great efforts. Here are some text suggestions.

**Review by hanxiaop — APPROVED** [review]
Looks good, thanks!

**Review by windsonsea — APPROVED** [review]
lgtm

**Pull request merged**